mne_bids/tests/test_write.py

```diff
@@ -56,16 +57,29 @@
 # using the stable bids-validator and make a direct call of bids-validator
 # also: for windows, shell = True is needed to call npm, bids-validator etc.
 # see: https://stackoverflow.com/q/28891053/5201771
-shell = False
-bids_validator_exe = ['bids-validator']
-if platform.system() == 'Windows':
-    shell = True
-    exe = os.getenv('VALIDATOR_EXECUTABLE', 'n/a')
-    if 'VALIDATOR_EXECUTABLE' != 'n/a':
-        bids_validator_exe = ['node', exe]
+@pytest.fixture(scope="session")
+def _bids_validate():
```

Comment:
Can you make a one line docstring to capture efficiently what this function does? It would help future contributors to take this in at a glance. See [pep8](https://www.python.org/dev/peps/pep-0008/#documentation-strings)

> Docstrings are not necessary for non-public methods, but you should have a comment that describes what the method does.